net/https-dns-proxy/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=https_dns_proxy
+PKG_VERSION:=2016-05-08e
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL=https://github.com/aarond10/https_dns_proxy/
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=5953a6ed393f09faf2f7edb38542d1b594d762f0
+PKG_MAINTAINER:=Aaron Drew <aarond10@gmail.com>
+PKG_LICENSE:=MIT
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/https_dns_proxy
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=DNS over HTTPS proxy server
+  DEPENDS:=+libcares +libcurl +libev
+endef
+
+TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
```

Comment:
`-I$(STAGING_DIR)/usr/include` should already be there by default, so this line can be dropped.